include/libcyphal/transport/scattered_buffer.hpp

```diff
@@ -167,11 +165,18 @@ class ScatteredBuffer final
     /// @param length_bytes The number of bytes to copy.
     /// @return The number of bytes copied.
     ///
-    CETL_NODISCARD std::size_t copy(const std::size_t offset_bytes,
-                                    void* const       destination,
-                                    const std::size_t length_bytes) const
+    /// NOSONAR cpp:S5008 below currently unavoidable. Could be fixed if nunavut provides `cetl::byte*` support.
```

Comment:
It looks like we're using `cetl::byte*` in other places in this PR now. Does this comment still apply?

Reply:
Yes, this comment still applies. Public facing `ScatteredBuffer` still uses `void*` as a second parameter. Internal stuff (aka `detail`) is switched to `cetl::byte*`.